src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -13,7 +13,7 @@ namespace System.Diagnostics.ProcessTests
     public partial class ProcessTest : IDisposable
     {
         private const int WaitInMS = 100 * 1000; 
-        private const string CoreRunName = "corerun";
+        private const string CoreRunName = "corerun.exe";
```

Comment:
> it looks like our CI system only does windows validation, how is one suppose to ensure that everything works cross-plat?

That's about to change.  We now have xunit up and running on CoreCLR on Linux, and @mmitche just harnessed it in the CI system.   We'll be turning it on for PRs once a few issues are sorted out (e.g. at the moment we have a lot of failures because tests that are Windows-specific haven't yet been annotated as such).